src/main.js

```diff
@@ -38,6 +39,7 @@ function TimekitBooking() {
   var rootTarget;
   var calendarTarget;
   var bookingPageTarget;
+  var testRibbonTarget;
```

Comment:
No need to make it a global class variable here as it is only used once in your render function (and besides, it is reinitialized on line 321 anyway)

Reply:
i see. good point